pages/patterns/delete-pattern.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
```

Comment:
Tempted to remove the redundant `-pattern` from the file name. As it's in the patterns folder, you should know its a pattern. Plus `/patterns/delete` is a nicer url